discovery-provider/nginx_conf/nginx.conf

```diff
@@ -65,9 +77,14 @@ http {
         location ~* .*/trending/.* {
             access_by_lua_block {
                 local main = require "main"
+                ngx.shared.request_count:incr("request-count", 1, 0)
                 return main.limit_to_rps()
             }
 
+            log_by_lua_block {
+                ngx.shared.request_count:incr("request-count", -1, 0)
```

Comment:
so when we process logs [which is the last step](https://johnzengtech.blogspot.com/2018/12/nginxopenresty.html) in request processing by nginx openresty, we clean up by decrementing request count which is a reference to # requests currently processing yes? if this is the case should we be incrementing in `init_by_lua` which is the entrypoint? just trying to clarify to expand my knowledge here.

Reply:
Yes, that's right, making these changes